### PR TITLE
Update defend-dos-attack.md

### PR DESCRIPTION
**Pull request by lwr20**
Fix GNS label

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: security/defend-dos-attack.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ kind: GlobalNetworkSet
 metadata:
   name: dos-mitigation
   labels:
-    dos-deny-list == 'true'
+    dos-deny-list: 'true'
```
Comment (neiljerram) [inline security/defend-dos-attack.md:76]: Don't really want the quotes there.  Does YAML remove them when reading this input?

Comment (lwr20) [reply security/defend-dos-attack.md:76]: Without the quotes, kubectl fails to apply this.  Calicoctl succeeds however.

```
lance@lwr20:~/scratch/goatseed-puppy$ kubectl apply -f - <<EOF
apiVersion: projectcalico.org/v3
kind: GlobalNetworkSet
metadata:
  name: dos-mitigation
  labels:
    dos-deny-list: true 
spec:
  nets:
  - "1.2.3.4/32"
  - "5.6.0.0/16"
  - "10.128.1.116"
EOF
error: unable to decode "STDIN": resource.metadataOnlyObject.ObjectMeta: v1.ObjectMeta.Labels: ReadString: expects " or n, but found t, error found in #10 byte of ...|ny-list":true},"name|..., bigger context ...|etworkSet","metadata":{"labels":{"dos-deny-list":true},"name":"dos-mitigation"},"spec":{"nets":["1.2|...
lance@lwr20:~/scratch/goatseed-puppy$ calicoctl apply -f - <<EOF
apiVersion: projectcalico.org/v3
kind: GlobalNetworkSet
metadata:
  name: dos-mitigation
  labels:
    dos-deny-list: true
spec:
  nets:
  - "1.2.3.4/32"
  - "5.6.0.0/16"
  - "10.128.1.116"
EOF
Successfully applied 1 'GlobalNetworkSet' resource(s)
```

Comment (neiljerram) [reply security/defend-dos-attack.md:76]: OK, this is because of `true` being a boolean variable, so the quotes are in fact needed.

**Review by neiljerram — APPROVED** [review]
LGTM

**Pull request merged**